BE/src/Controllers/GetPackageControllers.ts

```diff
@@ -78,34 +79,42 @@ export const GetPackageViaIDController = asyncHandler(
 export const GetPackageRatingsViaIDController = asyncHandler(
     async (req: GetPackageRatingsRequest, res: GetRatingsForPackageResponse, next: NextFunction) => {
         const requestedPackageID = req.requestedId;
-        //your code here
 
-        //^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-        //should return back to here with some type that fits the schema as follows...
+        const pack = await PackageModel.findById(requestedPackageID);
+        console.log(pack);
+
+        let DNE = false;
+
+        if (!pack) {
+            DNE = true;
+            let responseMessage = "Package does not exist.";
```

Comment:
swap to const plz, also add type so we get an error if the invalid response messages change. The type basically acts as an enum.

```ts
const responseMessage: GetRatingsForPackageInvalidResponses = "Package does not exist."
```